Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java

```diff
@@ -775,8 +775,8 @@ private void addWiimoteSubSettings(ArrayList<SettingsItem> sl, int wiimoteNumber
     }
     else
     {
-      extension = new WiimoteProfileSetting(mGameID, wiimoteNumber - 4, Settings.SECTION_PROFILE,
-              SettingsFile.KEY_WIIMOTE_EXTENSION, defaultExtension);
+      extension = new WiimoteProfileStringSetting(mGameID, wiimoteNumber - 4,
```

Comment:
What's the deal with `wiimoteNumber - 4`?  `getWiimoteProfile` just adds 4 to it again, so why the subtraction?  (I do see `mProfileKey = SettingsFile.KEY_WIIMOTE_PROFILE + (padID + 1)` in the settings, but I'm still not sure why it's like that)

Reply:
It's due to different mappings from the native code.

https://github.com/dolphin-emu/dolphin/blob/5abae61a8c7e0e4ac03bb8d3031793037e87fdd3/Source/Android/app/src/main/res/values/strings.xml#L25-L27

It could probably be improved but that's outside of the scope of this PR.

Reply:
Ah, I also missed the comment a few lines up:

https://github.com/dolphin-emu/dolphin/blob/de353551475808a7c403ca97df196331c1c628cb/Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java#L764-L767

The naming seems counterintuitive to me (it'd make more sense if what's in `WiimoteProfile*Setting` was called `wiimoteNumber` and what's in `addWiimoteSubSettings` was called `padID` since then `padID` would be 4-7 for wiimotes and the wiimote number would be 0-3), but it's probably something that would be handled by a larger rework outside of this PR.